views/guide.lavender

```diff
@@ -170,12 +171,15 @@ block content
           h3#variables Variables
           p
             | Lavender views can be passed variables from the parent php application, 
-            | or they can be defined right in the template.
+            | or they can be defined right in the template. This just assigns a variable
+            | without outputting anything to the page.
           p
-            | This just assigns a variable without outputting anything to the page.
+            | Note: While we do write it with no space, it is still valid with and extra space as
+            code.language-markup - some_variable
+            | . Expressions in general are not whitespace sensitive.
             pre 
               code.language-jade.
-               - some_variable = 'this is stored in a variable'
+               -some_variable= 'this is stored in a variable'
```

Comment:
you should have a space before this `=`. theres a difference between `p= foobar` and `-variable = 'foobar'`